modules/microshift-kickstart-rpm-install.adoc

```diff
@@ -12,11 +12,11 @@ Use the Kickstart file provided with {microshift-short} to provision an RPM-base
 
 * You set up the {microshift-short} Kickstart file.
 * You know the [Activation Keys](https://console.redhat.com/insights/connector/activation-keys) and organization ID to activate your Red Hat subscription.
-* You have the information needed to set the `BOOTC_IMAGE_URL`, `AUTH_CONFIG`, and `REGISTRY_CONFIG` variables.
+* You have the information needed to set the required variables.
 
 [IMPORTANT]
```

Comment:
This is about the subscription, should it be attached to the subscription bullet?